src/EditorFeatures/CSharpTest/ChangeSignature/AddParameterTests.AddImports.cs

```diff
@@ -0,0 +1,148 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.ChangeSignature;
+using Microsoft.CodeAnalysis.Editor.UnitTests.ChangeSignature;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities.ChangeSignature;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.ChangeSignature
+{
+    public partial class ChangeSignatureTests : AbstractChangeSignatureTests
+    {
+        [WpfFact, Trait(Traits.Feature, Traits.Features.ChangeSignature)]
+        public async Task AddParameterAddsAllImports()
+        {
+            var markup = @"
+class C
+{
+    void $$M() { }
+}";
+
+            var updatedSignature = new[] {
+                new AddedParameterOrExistingIndex(
+                    new AddedParameter(
+                        null,
+                        "Dictionary<ConsoleColor, Task<AsyncOperation>>",
+                        "test",
+                        "TODO"),
+                    "System.Collections.Generic.Dictionary<System.ConsoleColor, System.Threading.Tasks.Task<System.ComponentModel.AsyncOperation>>")};
```

Comment:
trying to figure out what the test is doing.  why is both `Dictionary<ConsoleColor, Task<AsyncOperation>>` specified as well as `System.Collections.Generic.Dictionary<System.ConsoleColor, System.Threading.Tasks.Task<System.ComponentModel.AsyncOperation>>`?

Reply:
Filed https://github.com/dotnet/roslyn/issues/42734 to reorganize & clean up test helpers.